src/package/nuspec/Microsoft.TestPlatform.Portable.nuspec

```diff
@@ -216,6 +216,7 @@
     <file src="net451\$Runtime$\zh-Hant\Microsoft.VisualStudio.TestPlatform.Common.resources.dll"	target="tools\net451\zh-Hant\Microsoft.VisualStudio.TestPlatform.Common.resources.dll" />
     <file src="net451\$Runtime$\zh-Hant\Microsoft.VisualStudio.TestPlatform.ObjectModel.resources.dll"	target="tools\net451\zh-Hant\Microsoft.VisualStudio.TestPlatform.ObjectModel.resources.dll" />
     <file src="net451\$Runtime$\zh-Hant\vstest.console.resources.dll"	target="tools\net451\zh-Hant\vstest.console.resources.dll" />
+    <file src="net451\$Runtime$\Microsoft.Extensions.FileSystemGlobbing.dll"	target="tools\net451\Microsoft.Extensions.FileSystemGlobbing.dll" />
```

Comment:
Place above the resources, just for consistency

Reply:
Done